MacDown/Resources/Extensions/mermaid.init.js

```diff
@@ -22,9 +22,9 @@
     }
  
     var insertSvg = function(svgCode, bindFunctions){
-      dom.innerHTML = svgCode;
+      this.innerHTML = svgCode;
```

Comment:
What is `this` in this context?

Reply:
insertSvg.bind(dom) will bind this to dom variable, but ensure it is the current from the loop.
We were facing the famous scope issue, this thread explains it : https://stackoverflow.com/questions/750486/javascript-closure-inside-loops-simple-practical-example

Reply:
Oh man. That's why modern JavaScript uses `let` and `const`…

Reply:
Yep

Reply:
I did a quick and simple fix but if needed we can adapt

Reply:
seems this patch will not be merged. have we gotten another solution? The duplicate in document is ugly and annoying。

Reply:
@legoming This pull request has already been merged. It is not in the latest test release, but will be in the next.

Reply:
@legoming Get familiar with GitHub! 😆

Reply:
need study @franklinyu 😁 

@uranusjr any schedule to release next test build? I'm looking forward to it a lot.